LSPS6/README.md

```diff
@@ -0,0 +1,89 @@
+# LSPS6 LSP meta-data
+
+| Name    | `meta-data` |
+| ------- |---------------|
+| Version | 1             |
+| Status  | Draft         |
+
+LSPS6 requires [BOLT8][] as a transport layer.
+
+[BOLT8]: https://github.com/lightning/bolts/blob/master/08-transport.md
+
+## Motivation
+
+Sometimes the client wants to obtain extra information from the LSP.
+There might be information that is not available through regular LSPS messages.
+An example for such information could be the pricing strategy for LSPS1.
+This extra information can be in the form of a URL or an email address.
+We need a way to prevent spoofing.
+
+### Actors
+
+The 'LSP' is the API provider, and acts as the server.
+The 'client' is the API consumer.
+
+## DNS Public Key Verification
+
+[BIP353][] provides a way to discover payment instructions over DNS.
+Analogous to this the LSP SHOULD create a DNS TXT record in the form of
+`lsps.acme.com. 3600 IN TXT "public_key:0326e6...2833a3"` for public key verification.
+
+[BIP353]: https://github.com/bitcoin/bips/blob/master/bip-0353.mediawiki
```

Comment:
Elaborate more on this. How is the TXT record structured. What is it used for? What do the numbers/letters after `public_key` mean? Is `public_key` mandatory? Is the public key in zbase32 or hex format? In a spec like this, you describe everything so it is idiot proof. Is any domain name allowed here or does it need to start with `lsps`? How do I distinguish this from other TXT records? Can I have multiple BIP353 records for the same domain? According to BIP353, this isn't invalid.

Also, rationales are needed why you want to do this. Lot's of open questions!